formats/indent.js

```diff
@@ -3,8 +3,8 @@ import Parchment from 'parchment';
 class IdentAttributor extends Parchment.Attributor.Class {
   add(node, value) {
     if (value === '+1' || value === '-1') {
```

Comment:
I actually think this can and should be removed... the +1/-1 is really a function of the toolbar and is handled as such in modules/toolbar.js, with the debugger attached, this method never got hit with a value that was a string === "+1" || "-1" it was always the existing indent incremented or decremented.